media-source-respec.html

```diff
@@ -2195,9 +2280,9 @@ <h2><dfn data-export="">Byte Stream Formats</dfn></h2>
               <p class="note">For example, if the first [=initialization segment=] has 2 audio tracks and 1 video track, then all [=initialization segments=] that follow it in the byte stream MUST describe 2 audio tracks and 1 video track.</p>
             </li>
             <li>[=Track IDs=] are not the same across [=initialization segments=], for segments describing multiple tracks of a single type (e.g., 2 audio tracks).</li>
-            <li>
-              <p>Codecs changes across [=initialization segments=].</p>
-              <p class="note">For example, a byte stream that starts with an [=initialization segment=] that specifies a single AAC track and later contains an [=initialization segment=] that specifies a single AMR-WB track is not allowed. Support for multiple codecs is handled with multiple <a>SourceBuffer</a> objects.</p>
+            <li><!-- BIG TODO verify this is good -->
```

Comment:
oops - I accidentally left this note to self in the merge commit. I'll update this PR to remove it.